Packs/Jira/Integrations/JiraV2/JiraV2.py

```diff
@@ -417,7 +479,12 @@ def get_issue_fields(issue_creating=False, **issue_args):
     if issue_args.get('assignee'):
         if not issue['fields'].get('assignee'):
             issue['fields']['assignee'] = {}
-        issue['fields']['assignee']['id'] = issue_args['assignee']
+        issue['fields']['assignee']['name'] = issue_args['assignee']
+
+    if issue_args.get('assignee_id'):
+        if not issue['fields'].get('assignee'):
+            issue['fields']['assignee'] = {}
+        issue['fields']['assignee']['accountId'] = issue_args['assignee_id']
```

Comment:
shouldn't `accountId` suppose to be `id`?

Reply:
Both are acceptable by the API but In the migration article, they're using the `accountId` (instead of the `id` as defined in the API docs) and found another issue where the usage of accountId was safer.